kernel/calls.c

```diff
@@ -253,7 +253,8 @@ void handle_interrupt(int interrupt) {
             deliver_signal(current, SIGSYS_, SIGINFO_NIL);
         } else {
             if (syscall_table[syscall_num] == (syscall_t) syscall_stub) {
-                printk("%d stub syscall %d\n", current->pid, syscall_num);
+                printk("%d(%s) stub syscall %d\n",current->pid, current->comm, syscall_num);
+
```

Comment:
```suggestion
```

Reply:
Ugh.  I can't believe I screwed up a one line PR.

Reply:
It's not a big deal, just delete the line, amend the commit, and push again :)